cyclonedx/output/xml.py

```diff
@@ -140,6 +140,73 @@ def _get_component_as_xml_element(self, component: Component) -> ElementTree.Ele
                 if len(ext_ref.get_hashes()) > 0:
                     Xml._add_hashes_to_element(hashes=ext_ref.get_hashes(), element=external_reference_e)
 
+        # releaseNotes
+        if self.component_supports_release_notes() and component.get_release_notes():
+            release_notes_e = ElementTree.SubElement(component_element, 'releaseNotes')
+            ElementTree.SubElement(release_notes_e, 'type').text = component.get_release_notes().get_type()
+            if component.get_release_notes().get_title():
+                ElementTree.SubElement(release_notes_e, 'title').text = component.get_release_notes().get_title()
+            if component.get_release_notes().get_featured_image():
+                ElementTree.SubElement(release_notes_e,
+                                       'featuredImage').text = str(component.get_release_notes().get_featured_image())
+            if component.get_release_notes().get_social_image():
+                ElementTree.SubElement(release_notes_e,
+                                       'socialImage').text = str(component.get_release_notes().get_social_image())
+            if component.get_release_notes().get_description():
+                ElementTree.SubElement(release_notes_e,
+                                       'description').text = component.get_release_notes().get_description()
+            if component.get_release_notes().get_timestamp():
+                ElementTree.SubElement(release_notes_e,
+                                       'timestamp').text = component.get_release_notes().get_timestamp().isoformat()
+            if component.get_release_notes().get_aliases():
+                release_notes_aliases_e = ElementTree.SubElement(release_notes_e, 'aliases')
+                for alias in component.get_release_notes().get_aliases():
+                    ElementTree.SubElement(release_notes_aliases_e, 'alias').text = alias
+            if component.get_release_notes().get_tags():
+                release_notes_tags_e = ElementTree.SubElement(release_notes_e, 'tags')
+                for tag in component.get_release_notes().get_tags():
+                    ElementTree.SubElement(release_notes_tags_e, 'tag').text = tag
+            if component.get_release_notes().get_resolves():
+                release_notes_resolves_e = ElementTree.SubElement(release_notes_e, 'resolves')
+                for issue in component.get_release_notes().get_resolves():
```

Comment:
method is getting to complex. put the body of the for/loop in an extra (private) method?